core/backend/src/test/standalone/GeometryStream.test.ts

```diff
@@ -2476,6 +2466,53 @@ describe("BRepGeometry", () => {
       assert(false, error.message);
     }
   });
+
+  it("catch solid kernel severe error - main thread", async () => {
+    const builder = new ElementGeometry.Builder();
+    builder.appendGeometryQuery(Loop.createPolygon([Point3d.create(0, 0, 0), Point3d.create(0, 1, 0), Point3d.create(-1.1, 1, 0), Point3d.create(-1, 1.1, 0)]));
+
+    // Test main thread using createBRepGeometry...
+    const onResult: BRepGeometryFunction = (_info: BRepGeometryInfo): void => { };
+
+    const createProps: BRepGeometryCreate = {
+      operation: BRepGeometryOperation.Offset,
+      entryArray: builder.entries,
+      onResult,
+      parameters: { distance: 0.25 },
+    };
+    try {

```

Comment:
```ts
expect(() => imodel.createBRepGeometry(createProps)).to.throw(ErrorType, "error message here");
```
(whatever the expected error type and error message are expected to be)

Reply:
Pushed...is Error the correct "ErrorType"?

Reply:
if you look at it in the debugger you'll see it's type. My guess is that it is just Error.